files/ko/learn/javascript/first_steps/strings/index.md

```diff
@@ -44,118 +44,9 @@ translation_of: Learn/JavaScript/First_steps/Strings
 
 문자열은 숫자와 유사하게 다루어지지만, 더 깊게 파고들면 눈에 띄는 차이점을 발견하기 시작할 것입니다. 먼저 몇 가지 기본 라인을 콘솔에 입력하여 우리와 친숙하게 만드는 것으로 시작하겠습니다. [이 링크](https://mdn.github.io/learning-area/javascript/introduction-to-js-1/variables/index.html)를 통해 다른 탭이나 창에서 열 수 있고, [브라우저 개발자 도구](/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools)를 사용할 수도 있습니다).
 
-<div class="hidden"><h6 id="Hidden_code">Hidden code</h6><pre class="brush: html">&#x3C;!DOCTYPE html>
```

Comment:
Looks like the conversion from HTML had a problem, so copied the current from the content repo